### PR TITLE
Spectral correlation density

### PR DESCRIPTION
**Pull request by shilpiprd**
ENH 
I did some research and the following code seemed like an estimation of Spectral Correlation Density. I got this code by looking in one of the links mentioned above.Not sure if it's correct.
If it's not I'd like to know if we're expected to write the entire code by ourself because i can't find anything more relevant.
Please review this is my first pull request. 

#14783 
I'm sorry, it seems I've submitted the same file twice

### CODE REVIEW

File: spectral.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+function Coh = SCoh_W(y,x,alpha,nfft,Noverlap,Window,opt,P)     
+% Coh = SCoh_W(y,x,alpha,nfft,Noverlap,Window,opt,P)
+% Welch's estimate of the Cyclic Spectral Coherence of signals y and x
```
Comment (shilpiprd) [inline spectral.txt:3]: It says Welch's estimation here and apparently Welch's method is one of the ways to create an estimation.

**Comment by rkern** [comment]
It looks like you just checked in the MATLAB code from MATLAB Central. That should just be a reference for you as you write the Python implementation to submit as a PR. We don't need the MATLAB code to be checked in. I suggest starting with a fresh branch (rather than `git rm`ing the MATLAB file and adding more commits to this one).

**Comment by tupui** [comment]
Hi @shilpiprd, thanks for this first draft. Aside from what Robert said, don't forget to add the license in a comment. Better solution would be to contact the original author and ask explicitly for use in SciPy under our BSD-3-clause.

https://www.mathworks.com/matlabcentral/fileexchange/48909-cyclic-spectral-analysis?s_tid=srchtitle#license_modal

**Comment by tupui** [comment]
Thank you again @shilpiprd. Since there was no progress here and lot of people seem interested, I will close this one in favour of #15519.

My message on the issue:

> Thank you for the new PR @qbarthelemy. Since @shilpiprd did not reply for a while either here and on the PR it's ok but we generally don't like to have competing PRs. I will close your PR @shilpiprd. You are all invited to help review/discuss the new PR https://github.com/scipy/scipy/pull/15519

**Pull request closed without merging**